Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7
+
+RUN apt -y update
+RUN apt -y install git
```

Comment:
you want to chain the update and install commands together, else caching can mess up by installing old versions.

`RUN apt -y update && apt -y install git`